File: cs/Hello.cs
// -*- coding: utf-8-unix; mode: csharp; -*-

//  SETUP
//    install mono-complete
//      $ sudo apt-get install mono-complete
//
//  COMPILE
//    $ mcs Hello.cs
//
//  RUN
//    $ mono Hello.exe

using System;

class Hello{
    static void Main(string[] args) {
        Console.Write("Hello, C#!");           // no new line
        Console.Write("\n");                   // new line
        Console.WriteLine("Good-bye, human!"); // with new line
        Console.WriteLine("こんにちは 世界!"); // 日本語も使える
        Console.WriteLine("円周率π={0,-2:F}", Math.Acos(-1));
    }
}

File: java/hello/Hello.java
package abc.java.hello;
// -*- coding: uft-8-unix; mode: java; -*-

/*
  SETUP
    $ sudo apt-get install default-jdk

  COMPILE
    $ javac Hello.java

  RUN
    $ java abc.java.Hello

  [eglot] Connected! Server `EGLOT (hello/(java-mode))' now managing `(java-mode)' buffers in project `hello'.

 */


import java.lang.Math;

class Hello {
    public static void main(String[] args) {
        System.out.print("Hello, Java!");
        System.out.print("\n");
        System.out.println("Good-bye, human!");
        System.out.println("こんにちは 世界!");
        System.out.printf("円周率π=%4.2f\n", Math.acos(-1));
    }
}
